apps/namadillo/jest.config.ts

```diff
@@ -1,9 +1,28 @@
-import { createDefaultPreset, pathsToModuleNameMapper } from "ts-jest";
+import { pathsToModuleNameMapper } from "ts-jest";
 import { compilerOptions } from "../../tsconfig.base.json";
 
 /** @type {import('ts-jest').JestConfigWithTsJest} */
 module.exports = {
-  ...createDefaultPreset(),
+  transform: {
+    "^.+\\.tsx?$": [
+      "ts-jest",
+      {
+        diagnostics: {
+          ignoreCodes: [1343],
```

Comment:
could you add some reference why we need this?

Reply:
I can try xD